src/router.ts

```diff
@@ -11,12 +11,13 @@ const routes: Array<RouteConfig> = [
   { path: '/:key/proposal/:id', name: 'proposal', component: Proposal },
   { path: '/:key/create', name: 'create', component: Create },
   { path: '/:key', name: 'proposals', component: Proposals },
+  { path: '/:key/:tab', name: 'proposals', component: Proposals },
   { path: '/', name: 'home', component: Home },
   { path: '/*', name: 'error-404', beforeEnter: (to, from, next) => next('/') }
 ];
 
 const router = new VueRouter({
-  mode: 'hash',
+  mode: 'history',
```

Comment:
There is a misunderstanding we don't want to change this otherwise it will break links, the server can not serve pages without hash like this http://snapshot.page/yam because its all static on IPFS, only index.html is served. What i were saying is that we should use `this.$route.param` to get the url parameters instead of doing string manipulation of the address hash.

Reply:
i see, updated